src/builtins.js

```diff
@@ -115,17 +115,9 @@ export default {
    * @returns {Object}
    */
   namedPipeline: function(name, args) {
-    const namedPipelineVar = 'namedPipelineOutput';
     return {
-      action: 'literal',
-      args: {
-        items: [ `%%vars.${namedPipelineVar}` ]
-      },
-      let: {
-        [namedPipelineVar]: {
-          '%pipeline': { name, args }
-        }
-      }
+      action: 'namedPipeline',
+      args: { name, args }
```

Comment:
for consistency, we should add in `service: ''` since the empty string represents the built-in service